TravelPoints-UI/src/app/services/auth.service.ts

```diff
@@ -19,19 +19,13 @@ export class AuthService {
         password: password
       }
     )
-    const headers = {'Content-Type': 'application/json'};
-    return this.httpClient.post<string>(this.baseURL + "/login", body, {headers});
+    return this.httpClient.post<string>(this.baseURL + "/login", body);
   }
   register(user: User): Observable<User> {
-    const headers = {'Content-Type': 'application/json'};
-    return this.httpClient.post<User>(this.baseURL + "/register", user, {headers});
+    return this.httpClient.post<User>(this.baseURL + "/register", user);
   }
 
   logOut(userId: number) {
-    let token = localStorage.getItem("token")
-    let headers = new HttpHeaders()
-      .set('Content-Type', 'application/json')
-      .set('Authorization', `Bearer ${token}`)
-    return this.httpClient.put<string>(this.baseURL + "/logout/" + userId, {headers});
+    return this.httpClient.put<string>(this.baseURL + "/logout/" + userId, null);
```

Comment:
S: don't put headers null, I think that you can just remove the parameter entirely